src/defrag/defrag.cpp

```diff
@@ -128,14 +127,21 @@ void Defrag::init()
 
 void Defrag::config()
 {
-
-	ASSERT( param(m_state_file,"DEFRAG_STATE_FILE") );
+#ifdef USE_DEFRAG_STATE_FILE
 	if( m_last_poll==0 ) {
 		loadState();
 	}
+#endif
+
+	if ( ! param(m_defrag_name,"DEFRAG_NAME")) {
+		param(m_defrag_name, "LOCALNAME");
+	}
+	auto_free_ptr valid_name(build_valid_daemon_name(m_defrag_name.c_str()));
+	ASSERT( valid_name );
+	m_daemon_name = valid_name.ptr();
 
 	int old_polling_interval = m_polling_interval;
-	m_polling_interval = param_integer("DEFRAG_INTERVAL",60);
+	m_polling_interval = param_integer("DEFRAG_INTERVAL",600);
```

Comment:
Did you intend to change this?  (You didn't mention it in the ticket.  It seems like a sensible change, though.)

Reply:
fixed to match the param table.